### PR TITLE
Fix docstring of Axes.cla()

### PR DESCRIPTION
**Pull request by timhoffm**
The docstring was written for `pyplot.cla()` but "Clear the current axes." is incorrect
for the Axes method. Also, `Axes.cla()` and `Axes.clear()` should have the
same docstring as they are synonyms.

To support a different docstring in pyplot, I had to explicitly define
`pyplot.cla()` and remove it from `boilerplate.py` generation. Given that
the function is trivial, this is an ok tradeoff for a better docstring
on the Axes.

### CODE REVIEW

**Review by jklymak — APPROVED** [review]
but a bit more invasive than a normal doc change, so prob should get second review to be safe...

**Review by dopplershift — APPROVED** [review]

**Pull request merged**
